core/os/os.h

```diff
@@ -356,6 +356,8 @@ class OS {
 	// This is invoked by the GDExtensionManager after loading GDExtensions specified by the project.
 	virtual void load_platform_gdextensions() const {}
 
+	virtual void alloc_console() {}
```

Comment:
```suggestion
	virtual void _alloc_console() {}
```

Since it's for internal use and not exposed.